src/azure-cli-testsdk/azure/cli/testsdk/base.py

```diff
@@ -3,29 +3,29 @@
 # Licensed under the MIT License. See License.txt in the project root for license information.
 # --------------------------------------------------------------------------------------------
 
-import unittest
 import os
-import inspect
 import subprocess
 import json
 import shlex
-import tempfile
-import shutil
 import logging
-import six
-import vcr
+import inspect
+import sys
+
+from azure_devtools.scenario_tests.base import LiveTest
+from azure_devtools.scenario_tests.base import ScenarioTest as DevToolsScenarioTest
+from azure_devtools.scenario_tests.recording_processors import (
+    SubscriptionRecordingProcessor, OAuthRequestResponsesFilter,
+    GeneralNameReplacer, LargeRequestBodyProcessor,
+    LargeResponseBodyProcessor, LargeResponseBodyReplacer,
+    DeploymentNameReplacer
+)
+from azure_devtools.scenario_tests.utilities import create_random_name
```

Comment:
I prefer all these names are exposed in one namespace instead of scattering in various namespaces, which hurts discoverability.